django_project/core/base_static/css/main.css

```diff
@@ -0,0 +1,220 @@
+body {
```

Comment:
Is this css for the accounts page? Maybe you can rename this file to accounts.css

Reply:
done @dimasciput